ddtrace/utils/install.py

```diff
@@ -0,0 +1,82 @@
+import sys
+
+from wrapt.importer import (
+    register_post_import_hook,
+    _post_import_hooks,
```

Comment:
I think I actually know the answer, but what versions of `wrapt` are all of these functions available for?

Since we don't lock `wrapt` to a specific version we might shoot ourselves in the foot if a customer either has an old version pinned, or a new version changes these.

Reply:
Yeah with this approach we'd have to lock wrapt.. which we should be doing anyways.

Reply:
I would love your review on #751, as it tried to vendor `six`.